database/transaction.go

```diff
@@ -103,6 +103,39 @@ func (tx Transaction) GetTable(name string) (*Table, error) {
 	}, nil
 }
 
+// RenameTable renames a table named old to new.
+// If it doesn't exist, returns ErrTableNotFound.
+func (tx Transaction) RenameTable(old, new string) error {
```

Comment:
We should avoid shadowing keywords, I'm surprised the linter didn't yell at you. Perhaps something like `oldName`, `newName` ?

Reply:
The linter didn't tell me anything regarding this. Does your tell you something?

Reply:
It doesn't, that's weird. Though it's a bad habit shadowing keywords, even if that sounds good in that context, plus being explicit in a function signature doesn't hurt